docs/data/whats-new.md

```diff
@@ -0,0 +1,20 @@
+# ✨ What's new in v6? ✨
+
+<p class="description">Discover the new features on the latest major version.</p>
+
+## Data Grid
+
+- [ApiRef available in the community version](https://next.mui.com/x/react-data-grid/api-object/)
```

Comment:
Maybe we should use relative paths like in most other places? 🤔 
```suggestion
- [ApiRef available in the community version](/x/react-data-grid/api-object/)
```

Reply:
Nice catch!